README.md

```diff
@@ -1 +1,45 @@
-# wcc-frontend
\ No newline at end of file
+<div style="display: flex; justify-content:center; padding-bottom: 50px">
+  <img src="public/logo_white.png" alt="WCC Logo White" width="200" height="200">
+</div>
+
```

Comment:
also this is nice: 

```
[![Contributors][contributors-shield]][contributors-url]
[![Forks][forks-shield]][forks-url]
[![Issues][issues-shield]][issues-url]
[![Stargazers][stars-shield]][stars-url]
```

Reply:
never seen or used that before 😅 - but we can add that later. What's the URL for those? 🤔

Reply:
It will show in the README the amount of forks, stars, issues and etc. 

![image](https://github.com/Women-Coding-Community/wcc-frontend/assets/3664747/68eabbd0-bdb0-431c-91ee-7f0801342f89)

Reply:
added 👍